### PR TITLE
Create automated_updates.yaml

### PR DESCRIPTION
**Pull request by samsonasik**
Based on https://github.com/rectorphp/rector/pull/6197#issuecomment-824789639

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Could we make it part of existing build to avoid duplication https://github.com/rectorphp/getrector.org/blob/main/.github/workflows/production_build.yaml ?

**Comment by samsonasik** [comment]
@TomasVotruba implemented https://github.com/rectorphp/getrector.org/pull/305/commits/91315ec53bd9343d0317c6a66bde7f7a376dfbdd

File: .github/workflows/production_build.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 name: Production build
 
-on: push
+on:
+  push: null
+  schedule:
+        # https://crontab.guru/every-7-days
+        -   cron: "0 0 * * 0"
```
Comment (TomasVotruba) [inline .github/workflows/production_build.yaml:7]:
```suggestion
        # https://crontab.guru/daily
        -   cron: "0 0 * * *"
```

File: .github/workflows/production_build.yaml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 name: Production build
 
-on: push
+on:
+  push: null
+  schedule:
+        # https://crontab.guru/daily
+        -   cron: "0 0 * * *"
```
Comment (TomasVotruba) [inline .github/workflows/production_build.yaml:7]:
```suggestion
on:
    push: null
    schedule:
        # https://crontab.guru/daily
        -   cron: "0 0 * * *"
```

Comment (TomasVotruba) [reply .github/workflows/production_build.yaml:7]: space indent

**Review by TomasVotruba — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thanks :+1:

**Comment by JanMikes** [comment]
Wow, this is easy and clever hotfix, did not realise it can be done in such easy way, i was thinking about triggers from rector repo to this repo to trigger build 😄 good job

**Comment by JanMikes** [comment]
But w8, this pull request does not solve updating, this will just re-deploy the same version of website that is already running, WITHOUT updated rector dependency. 

You need to run `composer update` before and changes in composer.lock should be commited

**Comment by TomasVotruba** [comment]
cc @samsonasik

**Comment by samsonasik** [comment]
For new commit of composer.lock, it seems needs git diff first to check if there is any file change?

**Pull request merged**